CTP/README.md

```diff
@@ -122,40 +124,63 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
   
 - **SHELL**
   - Prepare
-	* Prepare at least two accounts (e.g., one account ``controller`` as controller, another account ``shell_instance1`` as test instance)
-	* Check out CTP for each account and configure environment variables for CTP ``controller`` and test instance ``shell_instance1``
-	
-	  ```
+        * For local test
+	   * Prepare one account to use for controller and test instance
+	   * CTP and test scenario should be checked out on it, and configure JAVA_HOME
+	   ```
+	   JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)
+	   ```
+	   * **Example** ``shell.conf`` for a single instance, scenario and test build:
+	   ```
+	   default.cubrid.<property> //configure parameters for cubrid.conf
+	   default.broker1.<property> //configure parameters for cubrid_broker.conf
+	   default.broker2.<property>
+	   ```
+	   ```
+	   # Define the path of test cases used for testing, it should be checked out on test node
+	   scenario=${HOME}/cubrid-testcases/shell
+	   ```
+	   ```
+	   # Define the URL of test build which will be used to test.
+	   # If this parameter is not set or commented out, CTP will execute testing without build installation.
+	   cubrid_download_url=http://127.0.0.1/download/CUBRID-10.1.0.6929-b049ba5-Linux.x86_64.sh
+	   ```
+        * For multiple instances test
+	   * Prepare at least two accounts for the multiple test instances (e.g., one account ``controller`` as controller, another account ``shell_instance1`` as test instance)
+	   * Check out CTP for each account and configure environment variables for CTP ``controller`` and test instance ``shell_instance1``
+	    ```
 	   // on controller node
 	   JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)

```

Comment:
as is:
// on controller node

to be:
\# on controller node

Change others accordingly, please.